docs/content/cli-commands/npm-install.md

```diff
@@ -29,10 +29,17 @@ common options: [-P|--save-prod|-D|--save-dev|-O|--save-optional] [-E|--save-exa
 
 ### Description
 
-This command installs a package, and any packages that it depends on. If the
-package has a package-lock or shrinkwrap file, the installation of dependencies
-will be driven by that, with an `npm-shrinkwrap.json` taking precedence if both
-files exist. See [package-lock.json](/configuring-npm/package-lock-json) and [`npm shrinkwrap`](/cli-commands/shrinkwrap).
+This command installs a package and any packages that it depends on. If the
+package has a package-lock, or an npm shrinkwrap file, or a yarn lock file, the
+installation of dependencies will be driven by that, respecting the following
+order of precedence:
+
+* `node_modules/.package-lock.json`
```

Comment:
does this mean that even if i have `package-lock=false` in npmrc, `rm -rf node_modules && npm install` will no longer always fetch the latest of everything?

Reply:
well if you `rm -rf node_modules` first then the file is getting deleted along with the folder so it will still behave the same 😊

Reply:
and if i `rm -rf node_modules/* && npm install`, as i always do, since i have the `node_modules` dir ignored by dropbox and i don't want to lose that setting by deleting the folder? (which won't remove dotfiles)

Reply:
right right, yeah in that case it might not work as before but hey, the good news is that `npm update` should provide that functionality out of the box in npm7 and you won't need that convoluted workflow anymore! 😁